src/components/Post.tsx

```diff
@@ -2,8 +2,14 @@ import { h } from 'preact';
 import FavoriteBorderOutlinedIcon from '@mui/icons-material/FavoriteBorderOutlined';
 import ChatBubbleOutlineOutlinedIcon from '@mui/icons-material/ChatBubbleOutlineOutlined';
 import ShareOutlinedIcon from '@mui/icons-material/ShareOutlined';
+import { IconButton } from '@mui/material';
+import Button from '@mui/material/Button'
+import { useState, useEffect } from 'preact/hooks';
+import Favorite from '@mui/icons-material/Favorite'
 import EditIcon from '@mui/icons-material/Edit';
 import DeleteIcon from '@mui/icons-material/Delete';
+import { getAllComments } from '../utils/apiCalls'
+import CommentList from '../components/comment-components/CommentList'
```

Comment:
Nitpicky thing but can you add `;` at the end of your statements?

Reply:
Yes, I can add that as well.